Tests/Editor/Bootstrap/BootstrapSettingsTests.cs

```diff
@@ -5,6 +5,7 @@
 using AmazonGameLiftPlugin.Core.BucketManagement.Models;
 using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
 using AmazonGameLiftPlugin.Core.Shared;
+using Editor.CoreAPI;
```

Comment:
Added an item to bug tracker for this. Looks like StateManager's namespace wasn't updated in the previous PR